pom.xml

```diff
@@ -94,16 +78,16 @@
 
 
         <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-core</artifactId>
-            <version>2.9.1</version>
+            <groupId>info.picocli</groupId>
+            <artifactId>picocli</artifactId>
+            <version>3.0.0-alpha-4</version>
         </dependency>
 
 
         <dependency>
-            <groupId>info.picocli</groupId>
-            <artifactId>picocli</artifactId>
-            <version>3.0.0-alpha-4</version>
+            <groupId>life.qbic</groupId>
```

Comment:
You could and likely should use the postman-core-lib. The CLI is a standline tool. The library offers all utility functions for other tools.

Reply:
geo-uploader is now using postman core lib instead of the cli